replica/table.cc

```diff
@@ -1010,6 +1010,25 @@ void table::set_metrics() {
                     ms::make_gauge("cache_hit_rate", ms::description("Cache hit rate"), [this] {return float(_global_cache_hit_rate);})(cf)(ks)
             });
         }
+    } else {
+        if (_config.enable_node_metrics_reporting && !is_internal_keyspace(_schema->ks_name())) {
+            _metrics.add_group("column_family", {
```

Comment:
table?

Reply:
it should be compatible with the per-shard metrics